.github/workflows/node.yaml

```diff
@@ -0,0 +1,21 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Let's move to a [matrix for the OS too](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#configuring-a-build-matrix) in a follow-up, and add [Windows and macOS](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on)